plugins/inputs/redfish/README.md

```diff
@@ -0,0 +1,165 @@
+# Redfish Input Plugin
+
+The `redfish` plugin gathers  metrics and status information about CPU temperature, fanspeed, Powersupply, voltage, hostname and Location details(datacenter,placement,rack and room) of Dell hardware servers for which redfish is enabled.
+And also metrics like CPU temperature,fanspeed, Powersupply and hostname metrics for HP Hardware server(redfish should be enabled).
+
+Note: Currently this plugin only supports DELL and HP servers.
+
+
+
+### Configuration
+
+```toml
+[[inputs.redfish]]
+## Server OOB-IP
+host = "http://192.0.0.1"
+
+## Username,Password for hardware server
+basicauthusername = "test"
+basicauthpassword = "test"
+## Server Vendor(dell or hp)
+server= "dell"
+## Resource Id for redfish APIs
+id="System.Embedded.1"
+## Optional TLS Config
+# tls_ca = "/etc/telegraf/ca.pem"
+# tls_cert = "/etc/telegraf/cert.pem"
+# tls_key = "/etc/telegraf/key.pem"
+## Use TLS but skip chain & host verification
+# insecure_skip_verify = false
+
+## Amount of time allowed to complete the HTTP request
+# timeout = "5s"
+```
+
+### Metrics for Dell Servers
+
+- cpu_temperatures
```

Comment:
> The produced data model doesn't align very closely with what I see in the Redfish specification, I'd like to mirror the types as I think this will make it easier to extend to new types and understand for users familiar with Redfish. The most straightforward way to me seems to be to one metric for each supported resource type. For example we could have one type for Thermal, one for Power.

We have two type data coming from each resource(thermal,power) , if we send all the data from each resource to a single measurement it would be a problem as in the thermal resource  the number of fans and CPUs temperature varies, for example there might be 50 CPUs but only 7 Fans, Similar for Power resource as well. It would be appropriate to store them in different measurements like redfish_thermal_temperatures, redfish_thermal_fans,redfish_power_powersupplies, redfish_power_voltages. What would you think of it?